### DISCUSSION TITLE
[Feature] Configurable window for "On this day / X years since" reel

### DISCUSSION THREAD

**Discussion by thariq-shanavas** [immich-app/immich#6837] [Feature Request]
### The feature

Hello!

I love browsing the "X years since" photo reels on Immich. Unfortunately, I often miss these memory reels since I don't open the app every day.

I think a configurable window for how many days Immich considers for generating these memories would be fantastic 
e.g., generate albums for collections of pictures that were taken on this day +/- 3 days, in previous years.

<img width="862" alt="image" src="https://github.com/immich-app/immich/assets/19273195/14561f21-dec9-4a3d-acc3-64129491845a">

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Closing in favour of #15966
